package.json

```diff
@@ -7,11 +7,16 @@
     "start": "astro dev",
     "build": "astro build",
     "preview": "astro preview",
-    "astro": "astro"
+    "astro": "astro",
+    "format": "pnpm exec prettier . --write"
```

Comment:
Needs an ignore path passed to it; .gitignore should be fine

Reply:
are you saying that inside the quotes for format I need to add `.gitignore`?